src/Schema/Domain/Content/NameHelper.php

```diff
@@ -124,7 +153,7 @@ private function pluralize($name)
             return substr($name, 0, -2) . 'i';
         }
 
-        if (in_array(substr($name, -2), ['on', 'um'])) {
+        if (\in_array(substr($name, -2), ['on', 'um'])) {
```

Comment:
CS: the same remark as above

Reply:
Running `php-cs-fixer fix` drops `use function in_array;` and adds `\in_array` :man_shrugging: